Marlin/src/feature/pause.cpp

```diff
@@ -201,6 +201,9 @@ bool load_filament(const_float_t slow_load_length/*=0*/, const_float_t fast_load
 
     while (wait_for_user) {
       impatient_beep(max_beep_count);
+      #if BOTH(FILAMENT_CHANGE_RESUME_ON_INSERT, FILAMENT_RUNOUT_SENSOR)
+        if (READ(FIL_RUNOUT1_PIN) != FIL_RUNOUT1_STATE) wait_for_user = false;
```

Comment:
Is this load function still only applicable to the first filament sensor? The `MULTI_FILAMENT_SENSOR` option above implies that it should work for multiple filament sensors, so this could probably watch the runout pin that pertains to the `active_extruder` and not just the first one.

I expect this particular behavior will be rarely-used because most filament sensors are some distance from the extruder, and it usually takes a few tries to line up the filament with the opening to the Bowden tube. But it should be preverytty useful for setups where the filament path is reliable and the sensor is positioned within the extruder.

Reply:
It starts feeding before the material is there, which is fine. End result is either a couple extra purge presses or a little manual feed. On some extruders like the orbiter it's almost a necessity as there is no manual feed.

Reply:
The slight curve at the end of the filament is usually enough to thwart successful insertion on most of my printers. I usually hit the end of the filament with a heat gun or lighter to straighten it out before insertion, but often this isn't even enough to get reliable insertion into the tube. I wish more machines came with filament sensors between the drive wheel and the tube, and a nice funnel shape leading into the tube. Little touches like those could get us close to 100% reliability.